eml/tailwind.config.js

```diff
@@ -38,6 +38,7 @@ module.exports = {
       badgesPurple: '#C383F7',
       badgesBlue: '#54ADF1',
       unselectedStar: '#CBCAB8',
+      border: '#EAEAEA',
```

Comment:
if this border color should be universally applied to all borders in the app then the naming is fine, else it should probaly be more descriptive (eg primary-border if still the most common one).